tests/unit/test_client_gcs.py

```diff
@@ -0,0 +1,6 @@
+from datachain.client import Client
+
+
+def test_anon_url():
+    client = Client.get_client("gs://foo", None, anon=True)
+    assert client.url("bar") == "https://storage.googleapis.com/foo/bar"
```

Comment:
This test is going to hit gcs. I'd suggest mocking/stubbing `fs.sign` it.

Reply:
Thank you for suggestion! Done 👍